lib/rgeo/geographic/projected_feature_methods.rb

```diff
@@ -154,7 +167,7 @@ def self.included(klass)
 
       private
 
-      def validate_geometry
+      def prepare_geometry
```

Comment:
I wonder if I should change instances of `prepare_geometry` to another name. `prepare_geometry` is responsible for things like automatically closing linear rings or making sure that coordinates are on the globe, but it could be confused with functionality related to a prepared geometry (https://libgeos.org/usage/c_api/#prepared-geometry). Since this method is private and is only used in constructors of geometries it might not matter much though.

Reply:
@keithdoggett maybe `init_geometry` and a comment ? :)